meetings/rrsagent.md

```diff
@@ -0,0 +1,208 @@
+
+The list of action items may be reviewed and modified during the meeting. Action items will be recorded as they are modified in `https://www.w3.org/yyyy/mm/dd-<channel>-actions` where where '`yyyy`' is the current year, '`mm`' is the current month, and '`dd`' is the current day.
+
+- `[rrsagent,] ACTION: <text>`
+
+  Adds an action item described by `<text>`, assigning it the next sequential number. This command need not be directly addressed to RRSAgent. The keyword '`ACTION`' is not case sensitive. Actions may not be assigned in '`/me`' messages.
+
+- `[rrsagent,] ACTION <number> = <newtext>`
+
+  Replaces the text of action item `<number>` with `<newtext>`. This need not be directly addressed to RRSAgent, but will be ignored in '`/me`' lines. The keyword '`ACTION`' must be in all-caps or all lowercase, '`action`'.
+
+- `[rrsagent,] ACTION- <number>`
+
+  Removes action item `<number>` from the list. This need not be directly addressed to RRSAgent. The keyword '`ACTION`' is not case sensitive.
+
+- `rrsagent, what [are the] action items?`
+- `rrsagent, what [are the] actions?`
+- `rrsagent, [please] show [the] action items`
+- `rrsagent, [please] show [the] actions`
+- `rrsagent, [please] list [the] action items`
+- `actions?`
+
+  Displays the list of action items. The first several requests must be addressed specifically to RRSAgent, as shown. The last request will also work in '`/me`' (third-person or out-of-band) form.
+
+- `rrsagent, [please] drop action <number>`
+
+  Removes action item `<number>` from the list. This form must be addressed specifically to RRSAgent.
+
+- `rrsagent, [please] ignore action items`
+- `rrsagent, [please] ignore actions`
+
+  Requests that RRSAgent stop tracking action items. This request must be addressed specifically to RRSAgent.
+
+- `rrsagent, [please] track action items`
+- `rrsagent, [please] track actions`
+
+  Requests that RRSAgent begin tracking action items. This request must be addressed specifically to RRSAgent. This is the default state when RRSAgent is invited to a channel.
+
+### Searching the Log {#search}
+
+- `rrsagent, bookmark`
+- `rrsagent, pointer`
+- `rrsagent, here`
+- `rrsagent, where am I?`
+
+  Returns a URI to the current location in the log.
+
+- `rrsagent, grep [-i] [first-last|max] <text>`
+- `rrsagent, search for [-i] [first-last|max] <text>`
+- `rrsagent, find [-i] [first-last|max] <text>`
+- `rrsagent, <text>?`
+
+  Searches the log for `<text>`. The text is interpreted as a perl regex; e.g., '`.`' matches any character, '`<char>+`' matches one or more consecutive occurrences of `<char>`, and '`<char>*`' matches any number of consecutive occurrences of `<char>`, etc. '`-i`' causes the comparison to be case insensitive. '`first`', '`last`', and '`max`' limit the reported results to the specified (first and last) results or to at most (max) results.
```

Comment:
@swickr I first thought that `first-last` was a typo instead of `first|last` (first or last), but testing the search command I could not get any of those working.

See my tests in: https://www.w3.org/2025/01/24-vivtest-irc